### ISSUE TITLE
Missing red keyboard focus styles in A11y View on sim specific controls

### ISSUE THREAD

**Issue by terracoda** [phetsims/chipper#760]
@zepumph and @jessegreenberg, I can't say when the highlights stopped or if they were ever present (sorry). I don't think this issue should block publication or even an RC. I just would like to put it on th radar in case it is a general issue.

**Sim specific controls are not getting keyboard focus highlights** 
<img width="940" alt="Screen Shot 2019-06-13 at 11 32 52 AM" src="https://user-images.githubusercontent.com/1902116/59421610-2c26d580-8dcf-11e9-87fc-bfbc512de636.png">

<img width="982" alt="Screen Shot 2019-06-13 at 11 28 50 AM" src="https://user-images.githubusercontent.com/1902116/59421391-ce928900-8dce-11e9-9ed6-b40ef84e5984.png">

<img width="936" alt="Screen Shot 2019-06-13 at 11 33 15 AM" src="https://user-images.githubusercontent.com/1902116/59421628-347f1080-8dcf-11e9-8839-db6f8cfbbcc0.png">

**Reset All Button & Sim Resources buttons are getting focus highlights**
<img width="952" alt="Screen Shot 2019-06-13 at 11 29 15 AM" src="https://user-images.githubusercontent.com/1902116/59421437-df42ff00-8dce-11e9-8e3d-4ff2caccf6c0.png">
<img width="944" alt="Screen Shot 2019-06-13 at 11 29 49 AM" src="https://user-images.githubusercontent.com/1902116/59421443-e10cc280-8dce-11e9-913b-60f503bee6db.png">

**Comment by jessegreenberg** [comment]
Perhaps this is always been an issue. The highlight is shown in the a11y-view with the `border` attribute which according to https://stackoverflow.com/questions/2460501/how-to-change-checkboxs-border-style-in-css doesn't work on certain elements. That issue recommends using the outline attribute instead.

**Comment by jessegreenberg** [comment]
Fixed with the above commits.
